packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -73,12 +73,10 @@ and want to associate all stacks in the `App` scope to `MyAssociatedApplication`
 ```ts
 const app = new App();
 const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplication', {
-    applicationName: 'MyAssociatedApplication',
-    description: 'Testing associated application',
-    stackProps: {
-        stackName: 'MyAssociatedApplicationStack',
-        env: {account: '123456789012', region: 'us-east-1'},
-    },
+  appAssociatorProps: [appreg.ApplicationAssociatorPropsInputFactory.getApplicationAssociatorPropsFromAppName('MyAssociatedApplication', {
+    stackName: 'MyAssociatedApplicationStack',
+    env: { account: '123456789012', region: 'us-east-1' },
+  }, 'Testing associated application')],
```

Comment:
No more positional arguments after a "props" argument please. This is hard to read. Props arguments should be the last argument, anything else optional you want to add should just go into the props.